[needs-docs] Replace "Label with" with "Value"

DelazJ opened the pull request:
and harmonize with symbology renderers (fixes #30159)

![image](https://user-images.githubusercontent.com/7983394/64470634-f0d64180-d146-11e9-9a1d-473ab543e11e.png)

## Description
<!-- Include below a few sentences describing the overall goals for this pull request (PR). If applicable also add screenshots.-->

## Checklist

<!-- Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.
-->

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `Fixes #11111` at the bottom of the commit message
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit
- [ ] I have evaluated whether it is appropriate for this PR to be backported, backport requests are left as label or comment

nirvn commented:
-1, this label is inaccurate/misleading as the associated output can be an expression.

You're trying to fix the wrong end of the problem ;) IMHO we need to harmonize this away from 'label with' and 'column' to 'Value' or 'Column/Value'.

The reason why the renderers say use column is that it originally only supported columns. A few centuries ago ;-P

m-kuhn commented:
In addition to what @nirvn said, if the new term still contains "Column" it should be replaced with "Field Name" for consistency.

nyalldawson commented:
+1 for "Value"

DelazJ committed:
[needs-docs] Use "Value" as standard label of the widget

to filter labeling and symbology (categorized, graduated)
instead of "Label with" and "Column"
